src/server/scripts/Spells/spell_quest.cpp

```diff
@@ -437,68 +437,24 @@ enum q11520Roots
     SPELL_SUMMON_RAZORTHORN_ROOT        = 44941,
 };
 
-    class spell_q11520_discovering_your_roots : public SpellScript
-    {
-        PrepareSpellScript(spell_q11520_discovering_your_roots);
-
-        void HandleDummy(SpellEffIndex /*effIndex*/)
-        {
-            if (GameObject* go = GetCaster()->FindNearestGameObject(GO_RAZORTHORN_DIRT_MOUNT, 20.0f))
-            {
-                GetCaster()->GetMotionMaster()->MovePoint(0, *go);
-                go->SetLootState(GO_JUST_DEACTIVATED);
-                GetCaster()->CastSpell(GetCaster(), SPELL_SUMMON_RAZORTHORN_ROOT, true);
-            }
-        }
-
-        void Register() override
-        {
-            OnEffectHitTarget += SpellEffectFn(spell_q11520_discovering_your_roots::HandleDummy, EFFECT_0, SPELL_EFFECT_DUMMY);
-        }
-    };
-
-class spell_quest_dragonmaw_race_generic : public SpellScript
+class spell_q11520_discovering_your_roots : public SpellScript
 {
-    PrepareSpellScript(spell_quest_dragonmaw_race_generic);
-
-    bool Load() override
-    {
-        _x = _y = _z = 0.0f;
-        return true;
-    }
-
-    SpellCastResult RelocateDest()
-    {
-        Unit* caster = GetCaster();
-        float o = Position::NormalizeOrientation(caster->GetOrientation() + frand(0.0f, 2 * M_PI));
-        float dist = frand(5.0f, 30.0f);
-        _x = caster->GetPositionX() + dist * cos(o);
-        _y = caster->GetPositionY() + dist * std::sin(o);
-        _z = caster->GetPositionZ() + frand(-10.0f, 15.0f);
-        GetSpell()->m_targets.SetDst(_x, _y, _z, 0.0f, caster->GetMapId());
-        return SPELL_CAST_OK;
-    }
+    PrepareSpellScript(spell_q11520_discovering_your_roots);
 
-    void ChangeDest(SpellEffIndex effIndex)
+    void HandleDummy(SpellEffIndex /*effIndex*/)
     {
-        PreventHitDefaultEffect(effIndex);
-        Unit* caster = GetCaster();
-        if (Creature* trigger = caster->SummonCreature(23356, _x, _y, _z, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 1500))
+        if (GameObject* go = GetCaster()->FindNearestGameObject(GO_RAZORTHORN_DIRT_MOUNT, 20.0f))
         {
-            trigger->CastSpell(trigger, GetSpellInfo()->Effects[effIndex].TriggerSpell, true);
-            if (GetSpellInfo()->Effects[effIndex].TriggerSpell == 41064)
-                trigger->CastSpell(trigger, 41284, true);
+            GetCaster()->GetMotionMaster()->MovePoint(0, *go);
+            go->SetLootState(GO_JUST_DEACTIVATED);
+            GetCaster()->CastSpell(GetCaster(), SPELL_SUMMON_RAZORTHORN_ROOT, true);
```

Comment:
You need to validate spells used in spellscripts

Reply:
that one isn't mine, I just removed the script that was used for dragonmaw races :P